### PR TITLE
[FRONT-178] Use identicons for default avatar image

### PR DESCRIPTION
**Pull request by juhah**
On top of #1135, this adds default identicon profile images for ETH users:

Screenshot 2020-12-02 at 15 18 33
Screenshot 2020-12-02 at 15 18 27

For other users, the default remains the same:
Screenshot 2020-12-02 at 15 18 54
Screenshot 2020-12-02 at 15 19 00

Uploading a custom will override these. See storybook for other examples!

### CODE REVIEW

File: app/src/shared/components/AvatarImage/Initials.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        />
+        <Value>
+            {children}
+        </Value>
+    </div>
+)
+
+const Initials = styled(UnstyledInitials)`
+    background-color: #97BAC0;
+    color: white;
```
Comment (tumppi) [inline app/src/shared/components/AvatarImage/Initials.jsx:32]: Maybe use `#ffffff` for consistency?

File: app/src/userpages/components/ProfilePage/ProfileSettings/index.jsx [hunk 2]
```diff
@@ -29,23 +30,15 @@ const Root = styled.div`
...
-const StyledAvatarCircle = styled(AvatarCircle)`
-    && {
-        margin-right: 1.5rem;
-        width: 72px;
-        height: 72px;
-        line-height: 5rem;
-        font-size: 2em;
-        overflow: hidden;
+    ${AvatarCircle} {
+         background: white;
```
Comment (tumppi) [inline app/src/userpages/components/ProfilePage/ProfileSettings/index.jsx:35]: Maybe `#ffffff` here too

**Review by tumppi — APPROVED** [review]
Nice, works as it should 👍

**Review by mondoreale — APPROVED** [review]
👍

**Pull request merged**